builder/Makefile

```diff
@@ -0,0 +1,71 @@
+KUBE_VERSION = 1.4.0-beta.8
+
+RELEASE_URL_PREFIX := https://storage.googleapis.com/kubernetes-release/release
+
+PACKAGE_REV_KUBELET := 0
+DIRNAME_KUBELET = kubelet-$(ARCH)-$(KUBE_VERSION)-$(PACKAGE_REV_KUBELET)
+
+PACKAGE_REV_KUBECTL := 0
+DIRNAME_KUBECTL = kubectl-$(ARCH)-$(KUBE_VERSION)-$(PACKAGE_REV_KUBELET)
+
+# ARCH_amd64 := x86_64
+# ARCH_arm64 := arm64
+
+all:
+	for component in kubectl kubelet ; do for arch in amd64 arm64 ; do $(MAKE) $$component-build ARCH="$$arch" ; done ; done
+
+kubectl-build:
+	$(MAKE) kubectl-setup
+	$(MAKE) build/src/$(DIRNAME_KUBECTL)/usr/bin/kubectl
+	$(MAKE) build-packages \
+	  NAME="kubectl" \
+	  DESC="kubectl: The Kubernetes command line tool for interacting with the Kubernetes API" \
+	  ARCH="$(ARCH)" \
+	  ITER="$(PACKAGE_REV_KUBECTL)" \
+	  DIRNAME="$(DIRNAME_KUBECTL)" \
+	  SRCDIRS="usr/bin"
+
+kubelet-build:
+	$(MAKE) kubelet-setup
+	$(MAKE) build/src/$(DIRNAME_KUBELET)/usr/sbin/kubelet
```

Comment:
It seems like `/usr/sbin` is the right place. I'd put `kubeadm` there also, as it has to run as root. What do people think?